include/deal.II/lac/constraint_matrix.templates.h

```diff
@@ -27,6 +27,7 @@
 #include <deal.II/lac/sparse_matrix.h>
 #include <deal.II/lac/block_sparsity_pattern.h>
 #include <deal.II/lac/block_sparse_matrix.h>
+
```

Comment:
Done. In future, this will be done by a script (I've seen the comment above to late).